source/common/common/version.h

```diff
@@ -2,7 +2,13 @@
 
 #include <string>
 
+#include "common/common/version_number.h"
+
 namespace Envoy {
+
+// Release version number (e.g. 1.6.0).
+static const std::string& VERSION_NUMBER = BUILD_VERSION_NUMBER;
```

Comment:
This is a global non-POD, which is not allowed in Envoy style, see point on " non-PoD static and global variables are forbidden" in https://github.com/envoyproxy/envoy/blob/master/STYLE.md#deviations-from-google-c-style-guidelines.

Is defining this necessary? Can't we just use the C string from `BUILD_VERSION_NUMBER` everywhere directly?

Reply:
Oops, I missed the style guide. I think we can use the `BUILD_VERSION_NUMBER` everywhere direcly. d6b7047d2